tests/channels/test_jpsi_to_gamma_pi0_pi0.py

```diff
@@ -17,30 +17,54 @@
 logging.getLogger().setLevel(logging.ERROR)
 
 
+@pytest.mark.parametrize(
+    "allowed_intermediate_particles, number_of_solutions",
+    [
+        (["f(0)(1500)"], 4),
+        (["f(0)(980)", "f(0)(1500)"], 8),
+        (["f(2)(1270)"], 12),
+        (["omega(782)"], 16),
```

Comment:
Can you make a small comment that "basic" number of solutions is 8, but due to identical particle combinatorics it get increased by a factor of 2.